bridge/svix-bridge/src/webhook_receiver/mod.rs

```diff
@@ -181,5 +197,161 @@ async fn transform(
+    const NO_SLEEP: Duration = Duration::ZERO;
+    let mut sleep_time = NO_SLEEP;
+
+    let PollerInputOpts::SvixEvents {
+        app_id,
+        subscription_id,
+        ..
+    } = &poller.input_opts;
+
+    let mut iterator = None;
+
+    loop {
+        tracing::trace!(app_id, subscription_id, "polling `/events`");
+        match poller
+            .svix_client
+            .message()
+            .events(V1MessageStreamParams {
+                // FIXME: expose more params as poller input cfg
+                app_id: app_id.clone(),
+                limit: None,
+                iterator: iterator.clone(),
+                event_types: None,
+                channels: None,
+                after: None,
+            })
+            .await
+        {
+            Ok(resp) => {
+                tracing::trace!(count = resp.data.len(), "got messages");
+                for msg in resp.data.into_iter() {
+                    let payload = match parse_payload(
+                        &SerializablePayload::Standard(
+                            // FIXME: for svix-event pollers we already know the payload is json so
+                            //   there's some wasted ser/deser/ser cycles.
+                            serde_json::to_vec(&msg)
+                                .expect("just fetched as json, must be serializable"),
+                        ),
+                        poller.transformation.as_ref(),
+                        poller
+                            .transformer_tx
+                            .clone()
+                            .expect("transformer tx is required"),
+                    )
+                    .await
+                    {
+                        Err(status) => {
+                            tracing::error!(
+                                status = status.as_u16(),
+                                "error while parsing polled message"
+                            );
+                            // BACKOFF
+                            sleep_time = (sleep_time * 2).clamp(MIN_SLEEP, MAX_SLEEP);
+                            // Retry the current iterator.
+                            continue;
+                        }
+                        Ok(p) => p,
+                    };
+                    if let Err(status) = handle(payload, poller.output.clone()).await {
+                        // FIXME: need to refactor handle to not give http status codes so we can report what happened here.
+                        tracing::error!(
+                            status = status.as_u16(),
+                            "error while handling polled message"
+                        );
+                        // BACKOFF
+                        sleep_time = (sleep_time * 2).clamp(MIN_SLEEP, MAX_SLEEP);
+                    }
+                }
+                tracing::trace!(
+                    iterator =? &iterator,
+                    next_iterator =? &resp.iterator,
```

Comment:
I find this spelling a bit obscure, though I guess I see its advantages. This should have exactly the same semantics and is how I usually see tracing fields written:

```suggestion
                    ?iterator,
                    next_iterator = ?resp.iterator,
```